### DISCUSSION TITLE
Different override ''tasks''

### DISCUSSION THREAD

**Discussion by iliadaili** [proginosko/LeechBlockNG#371] [Ideas]
Hi! I had another idea for a possible update. Instead of only having the option to lock override with a random code that you need to transcript, maybe there could be other things, like random math equations, puzzles, things like that. This is extremely unnecessary, I'm suggesting just in case, because I thought that, albeit unnecessary, it's a kinda neat idea

**Comment by xuiqzy** [comment]
A simple delay of x seconds (maybe not shown and with added randomness to the amount of delay) would also help in use cases where the delayed page feature cannot be used and one rather uses blocked pages with regular temporary overrides.

**Comment by crystalreef22** [comment]
I feel like a good idea would be to make it so that you need to hold the spacebar or hold a button for a specified amount of seconds before overriding
